### PR TITLE
Add italian README and patterns of README translations (and solve #471)

### PR DESCRIPTION
**Pull request by thaleseuflauzino**

### CODE REVIEW

**Comment by thaleseuflauzino** [comment]
this pr solve #471

File: README.pt-BR.md [hunk 1]
```diff
@@ -13,40 +13,43 @@
...
+  - [ Instale Dependencias do Node](#-instale-dependencias-do-node)
+  - [ Instale Python 3.9](#-instale-python-39)
+  - [ Instale Python Dependencies](#-instale-python-dependencies)
+- [ Environment variables](#-environment-variables)
+- [ Running](#-running)
+- [ Build](#-build)
+  - [ Build the bittorrent client](#-build-the-bittorrent-client)
+  - [ Build the Electron application](#-build-the-electron-application)
+- [ Contributors](#-contributors)
+- [ Licença](#-licença)
```
Comment (zamitto) [inline README.pt-BR.md:52]: There is a space after the "[" in a lot of lines, and in other files as well. Is it intentional?

**Comment by zamitto** [comment]
A German README has just been added, could you update your PR?

**Comment by thaleseuflauzino** [comment]
> A German README has just been added, could you update your PR?

sure

File: README.pt-BR.md (hunk 1 above)
Comment (thaleseuflauzino) [reply README.pt-BR.md:53]: i try to remove, but, may lint put the space again

**Review by thaleseuflauzino — COMMENTED** [review]
may now the spaces is correct

**Review by zamitto — APPROVED** [review]

**Pull request merged**